src/modules/playback-engine/output/native/adapters/native.ts

```diff
@@ -42,10 +42,6 @@ export default function getNativeAdapterCreator(
       this._bindCallbacks();
     }
 
-    get currentUrl() {
```

Comment:
Why you removing it?

Reply:
Editor shows no usage of this property.

Reply:
take a look on a second commit, pls. I removed _curentSrc from engine. Now we take it directly from output(adapters). Looks like this approach works fine, but still validate me on this pls.